logging/log_storage/installing-log-storage.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="installing-log-storage"]
+= Installing log storage
+:context: installing-log-storage
+
+toc::[]
+
+You can use the {oc-first} or the {product-title} web console to install a log store on your {product-title} cluster.
+
+include::snippets/logging-elastic-dep-snip.adoc[]
+
+// install Loki
+include::modules/cluster-logging-loki-deploy.adoc[leveloffset=+1]
+include::modules/logging-deploy-loki-console.adoc[leveloffset=+1]
+include::modules/logging-loki-gui-install.adoc[leveloffset=+1]
+include::modules/logging-loki-cli-install.adoc[leveloffset=+1]
+
+// install ES
```

Comment:
Can we have headings for the two sections, so that it's clear from the table-of-contents of this page that there are topics covering Loki and topics covering ES and they have to connection to each other?